build/ical.js

```diff
@@ -6668,6 +6681,416 @@ ICAL.Event = (function() {
   return Event;
 
 }());
+
+ICAL.Todo = (function() {
```

Comment:
Yes, I don't know that first must add a todo file and then do make package.

El 08/04/15 a las 08:43, Philipp Kewisch escribió:
>
> In build/ical.js 
> <https://github.com/mozilla-comm/ical.js/pull/147#discussion_r27952101>:
>
> > @@ -6668,6 +6681,416 @@ ICAL.Event = (function() {
> >    return Event;
> >
> >  }());
> > +
> > +ICAL.Todo = (function() {
>
> I assume a lot of this is copy/paste from ICAL.Event? If so, it would 
> probably make sense to create a BaseItem class. I don't think the 
> BaseItem class needs to be exported on the ICAL object unless its 
> needed to keep it in a separate file.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mozilla-comm/ical.js/pull/147/files#r27952101>.
>